Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:focal
+FROM nvidia/cuda:11.4.0-runtime-ubuntu20.04
```

Comment:
This docker image itself can be installed and run on macos, but the nvidia runtime (which I'm adding to the README) can't. 

So you can run this repo locally on macos, but standalone. In order to do that instead of running on docker, run it on a virtualenv and install the packages manually.